src/Stripe.net/Services/FileUploads/FileUploadService.cs

```diff
@@ -18,34 +18,35 @@ public FileUploadService(string apiKey)
         {
         }
 
-        public virtual FileUpload Create(string fileName, Stream fileStream, string purpose, RequestOptions requestOptions = null)
+        public virtual FileUpload Create(FileUploadCreateOptions options = null, RequestOptions requestOptions = null)
```

Comment:
wouldn't options be required for file upload creation?

Reply:
Yep. I'll fix.